packages/trpc/server/routers/viewer/availability/schedule/get.handler.ts

```diff
@@ -30,13 +30,31 @@ export const getHandler = async ({ ctx, input }: GetOptions) => {
       name: true,
       availability: true,
       timeZone: true,
+      eventType: {
+        select: {
+          _count: true,
```

Comment:
This resulted in what was the slowest query in our system. It was removed in #10842 because it wasn't used. These `_count: true` flags with no subfields supplied tell prisma to aggregate count all relation tables.